data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/OnFlowConstraintImpl.java

```diff
@@ -20,7 +20,7 @@ public class OnFlowConstraintImpl extends AbstractUsageRule implements OnFlowCon
     private FlowCnec flowCnec;
 
     OnFlowConstraintImpl(Instant instant, FlowCnec flowCnec) {
-        super(UsageMethod.TO_BE_EVALUATED);
+        super(UsageMethod.AVAILABLE);
```

Comment:
actually you have to add this as an argument + attribute, and take it from the Adder. Same or OnFlowConstraintInCountry, OnVoltageContraint, OnAngleConstraint